src/web/types/Abstractapptypes.ts

```diff
@@ -0,0 +1,10 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+export default interface Abstractapptypes {
```

Comment:
I think we should rename these interfaces to reflect what they actually represent.
E.g. AbstractPathParameters, DefaultPathParameters etc.